tasks/deploy.py

```diff
@@ -74,6 +75,26 @@ def determine_job_dirs(pr_number):
     return job_directories
 
 
+def determine_pr_comment_id(job_dir):
```

Comment:
It would probably be good to implement a test for this function, but I see that we don't have any tests for the functions in `tasks/deploy.py`, so this can be done later for all of them.